common/match/dissolve.ts

```diff
@@ -39,6 +39,12 @@ export async function dissolveMatch(match: Match, dissolveReason: number, dissol
 
     await Notification.actionTaken(student, 'tutor_match_dissolved', { pupil, matchHash, matchDate, uniqueId });
     await Notification.actionTaken(pupil, 'tutee_match_dissolved', { student, matchHash, matchDate, uniqueId });
+
+    if (dissolver.email === student.email) {
+        await Notification.actionTaken(pupil, 'tutee_match_dissolved_other', { student, matchHash, matchDate, uniqueId });
```

Comment:
What's the difference to the `tutee_match_dissolved` notification above?

Reply:
This one is only triggered for the other person of the match, i.e. when I resolve a match then only my match partner is notified (I don't need to be notified as I triggered the action)

Reply:
No, that depends on the notifications attached to the action.

Reply:
We will probably replace tutee_match_dissolved with tutee_match_dissolved_other in the current notification

Reply:
> No, that depends on the notifications attached to the action.

So then it's not a no, but a maybe 😅 



> We will probably replace tutee_match_dissolved with tutee_match_dissolved_other in the current notification

Okay, I see